app.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, render_template, url_for
+
+app = Flask(__name__)
+
+@app.route('/')
+def home():
+    return render_template('home.html')
+
```

Comment:
pep8 style is not implemented, between functions make sure that there is  2 empty lines, follow pycharm hints it's very helpful